pytripgui/plan_vc/plan_view.py

```diff
@@ -0,0 +1,263 @@
+from PyQt5 import QtWidgets
+from PyQt5.QtCore import Qt
+from PyQt5.QtWidgets import QTreeWidgetItem
+from pytripgui.view.gen.plan import Ui_PlanDialog
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+class PlanQtView(object):
+    """
+    """
+    def __init__(self):
+        self.ui = Ui_PlanDialog()
+        self.dialog = QtWidgets.QDialog()
+
+        self.ui.setupUi(self.dialog)
+        self._setup_callbacks()
+
+    def show(self):
+        self.dialog.show()
+        self.dialog.exec_()
+
+    def exit(self):
+        self.dialog.close()
+
+    def set_ok_callback(self, fun):
+        self.ui.accept_buttonBox.accepted.connect(fun)
+
+    def set_cancel_callback(self, fun):
+        self.ui.accept_buttonBox.rejected.connect(fun)
+
+    def _setup_callbacks(self):
+        pass
+
+    def get_basename_value(self):
+        return self.ui.basename_lineEdit.text()
+
+    def set_basename_value(self, basename):
+        self.ui.basename_lineEdit.setText(basename)
+
+    def get_comment_value(self):
+        return self.ui.comment_lineEdit.text()
+
+    def set_comment_value(self, comment):
+        self.ui.comment_lineEdit.setText(comment)
+
+    def get_uuid_value(self):
+        return self.ui.uuid_lineEdit.text()
+
+    def set_uuid_value(self, uuid):
+        self.ui.uuid_lineEdit.setText(uuid)
```

Comment:
(should never be set by user, FYI)

Reply:
I agree - UUID is generated automatically, see 
https://github.com/pytrip/pytrip/blob/master/pytrip/tripexecuter/plan.py#L88
```
self.__uuid__ = uuid.uuid4()  # for uniquely identifying this plan
```

Reply:
Of course. But user shouldnt use view class.
set_uuid_value is part of public interface of view.
controller call set_uuid_value to set uuid in view.
Thre is no interface to set uuid in model

Reply:
After comment of @ljelen I think it makes more sense to have it in the controller/view. In such case it could be used i.e. in debugging printouts / displays, even if we do not implement them now.

Reply:
There is no problem with debugging such a code.
with qtbot you can talk direcly to qt window, so i propose that flow:
1. set UUID in model to known value
2. connect model and view with controler
3. call 'set_view_from_model' method
4. check with qtboot is UUID is valid(directly on widnow)